src/assets/Standards/CollectibleStandards/ERC1155/ERC1155Standard.ts

```diff
@@ -188,6 +188,10 @@ export class ERC1155Standard {
     image: string | undefined;
   }> => {
     const isERC1155 = await this.contractSupportsBase1155Interface(address);
+
+    if (!isERC1155) {
```

Comment:
What motivated this change? Was this just for performance reasons, or did something blow up here before this was done.

Reply:
I assumed this was to mirror the changes in ERC721

Reply:
Mainly I wanted to ensure that if this fixed something, we had that captured in a unit test. I do think this is a good change regardless.

Reply:
It doesn't fix anything, just potentially reduces unnecessary network requests.